content/post/practical_ml_2.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Ilan Man
+title: "More Tips for Practical Data Science"
+cover: ""
+publishDate: TBD
+draft: false
+tags:
+ - datascience
+---
+
+This is the second installment of my "Tips on Practical Data Science". In today's post we're going to discuss implications of a changing data distribution, some practical technical considerations, and the value of presenting your work. 
```

Comment:
Great intro. I'd suggest adding a link to the previous post (and modifying the previous post to have a link to the new one)